preprocessors/charts.schema.json

```diff
@@ -0,0 +1,51 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/charts.schema.json",
+    "type": "object",
+    "title": "Charts preprocessor",
+    "description": "Extracts additional information from the datapoints present in the graph",
+    "properties": {
+        "dataPoints": {
+            "type": "array",
+            "title": "dataPoints",
+            "description": "The dataPoints in the graph",
```

Comment:
`title` and `description` are purely annotations. I'm not sure why you write it as "dataPoints", does this have a special meaning elsewhere?